explainer.md

```diff
@@ -505,10 +505,10 @@ uploaded to the server but have been cached locally for easy user access. The
 have not yet but uploaded to the server and are irrecoverable if lost.
 
 In this scenario, the `drafts` bucket is created with `persisted: true` and
-`durability: "strict"` to specify that it should be evicted last upon storage
+`durability: "strict"` to specify that it should not be evicted upon storage
 pressure, and all data should survive power failures at the cost of more
-battery consumption. The `recent` bucket will be evicted first because of its
-low priority since it contains data that can be recovered from the server.
+battery consumption. The `recent` bucket will be evicted upon storage pressure because
```

Comment:
Somewhat tangential, but now seems like a good time to update this part:

"""
`durability: "strict"` to specify that it should not be evicted upon storage
pressure, and all data should survive power failures at the cost of more
battery consumption.
"""
Strict durability neither grants this guarantee, nor does the lack of it mean that data is automatically lost during power failures. Let's not suggest its usage.

I think this example also confuses these two policies because it makes it sound like "important data should always be strict and persisted, unimportant data should always be relaxed and non persisted", which presents the question, "why there are two different policies?"

Also, nit: "because of its low priority" makes it sound like there's a policy called `priority`, but since there isn't, I would have expected this to be phrased "because it is marked `persisted`" or to perhaps reword the entire sentence as "The recent bucket is deemed lower priority, so it can be evicted on storage pressure, and thus persistence is not requested." or something like that.

Reply:
Updated text to address comment, let me know if anything is missing or if I should rephrase. Thanks!

Reply:
My suggestion would be to avoid using this example to illustrate `durability` at all. The rest of it LGTM.

I filed https://github.com/WICG/storage-buckets/issues/108 for the durability issue. I actually can't think of any great reason to mark an entire bucket as durable, although that's not to say it doesn't exist.

Reply:
Sure, removed from this example.